doc/org/build/networking.md

```diff
@@ -0,0 +1,256 @@
+- [Intro](#orgca5b855)
+- [The `Networked` Component](#orgabca6d0)
+- [Message Types](#orge720d3c)
+  - [`CreateMessage`](#org4f63df9)
+  - [`UpdateMessage`](#orge698598)
+  - [`DeleteMessage`](#org8f17ac8)
+  - [`ClientJoin`](#orgf98374b)
+  - [`ClientLeave`](#orgebe8740)
+  - [`CreatorChange`](#org9b3f8a5)
+- [Eventual Consistency](#orga8173d0)
+- [Creating Networked Entities](#org62cc95d)
+- [Writing Networked Components](#orgd90eddd)
+- [Persisting Networked Entity State](#orgd0901c8)
+- [Networked Entity Hierarchies](#orge3d34eb)
+
+
+<a id="orgca5b855"></a>
+
+# Intro
```

Comment:
Similar to gameplay document, I would like to suggest to start with a simple and workable example, and then complex example like the media one in the later section.